apps/condo/domains/organization/sbbol/oauth2.js

```diff
@@ -0,0 +1,170 @@
+// Todo(zuch): need to write JWT verification
+
+const conf = process.env
+
+const { Issuer, custom } = require('openid-client') // certified openid client will all checks
+const jwtDecode = require('jwt-decode') // decode jwt without validation
+
+const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
+const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
+const JWT_ALG = 'gost34.10-2012'
```

Comment:
If we are not exporting this variables, we can move them to `SbbolOauth2Api` constructor

Reply:
Maybe, later. @AntonAL will continue to work with sbbol - maybe, he will move it to .env or constants